### PR TITLE
[EDA]  People & Groups: Account Model Settings First Page

### PR DESCRIPTION
**Pull request by m-kinkade**
# Critical Changes

# Changes

# Issues Closed

# New Metadata
### LWC Component
accountModelSettings

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: cumulusci.yml [hunk 1]
```diff
@@ -281,6 +281,15 @@ tasks:
             path: unpackaged/config/trial
             namespace_tokenize: $project_config.project__package__namespace
 
+    deploy_Summer_21QA:
+        group: "EDA Metadata"
+        description: Deploys QA and Dev Only Config for Summer ‘21
+        class_path: cumulusci.tasks.salesforce.DeployBundles
+        options:
+            namespace_inject: $project_config.project__package__namespace
```
Comment (davidmreed) [inline cumulusci.yml:289]: The `namespace_inject` and `unmanaged` options are no longer required. CumulusCI can infer the needed configuration now!

**Review by davidmreed — APPROVED** [review]

File: cumulusci.yml (hunk 1 above)
Comment (MatthewBlanski) [reply cumulusci.yml:296]: Thank you. This is fixed up in another branch. We have some interesting concurrent branch strategies at the moment that will exist only for this set of functionality.

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]
Two tiny changes, then we're off to the approval races!

**Review by MatthewBlanski — APPROVED** [review]

**Comment by davidmreed** [comment]
`prettier` is showing some format issues in the lint task.

**Comment by spinkelman** [comment]
This item is untestable until it's merged with the feature/232__saveFunctionality branch. Merging to test.

**Pull request merged**